cmd/startup/startup.go

```diff
@@ -365,7 +365,7 @@ func startQueryServer(serverAddr string) {
 					return emptyHtmlContent
 				},
 				"CSSVersion": func() string {
-					return "0.0.1"
+					return "0.2.32"
```

Comment:
Should we change this to `0.0.2` or the same version as SigLens: `0.2.32`?

Reply:
keep it the same as the release version